tlsn/tlsn-core/src/session/mod.rs

```diff
@@ -21,6 +21,7 @@ use crate::{
     Commitment, Direction, InclusionProof, SubstringsCommitment, SubstringsCommitmentSet,
 };
 
+/// A potentially notarized session
```

Comment:
Only prover deals with `NotarizedSession`. From his pov, it is not "potentially" correct, it is definitely correct. Maybe better "A validated notarized session stored by the Prover"